aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -614,6 +623,11 @@ public static ModifyDbInstanceRequest modifyDbInstanceAfterCreateRequest(final R
             builder.storageThroughput(model.getStorageThroughput());
             builder.storageType(model.getStorageType());
         }
+
+        if (ResourceModelHelper.isOracleCDBEngine(model.getEngine())) {
+            builder.engine(model.getEngine());
```

Comment:
Is this parameter idempotent? What I mean is, if the instance being restored already had the CDB engine, and the Modify call includes the same CDB engine, will it succeed on the RDS side without making any changes or will it fail with a validation error? We have seen cases of non-idempotent behavior in the past, so we need to take that into consideration.

Reply:
it's idempotent, I tried that manually. If there are no other changes though, there will be no modify after create as CDB to CDB is not conversion and will be skipped from CFN side.